data/linux-bridge/002-linux-bridge.yaml

```diff
@@ -35,14 +35,15 @@ spec:
             - /bin/bash
             - -c
             - |
-              cp -rf /usr/src/containernetworking/plugins/bin/*bridge /opt/cni/bin/
-              cp -rf /usr/src/containernetworking/plugins/bin/*tuning /opt/cni/bin/
+              echo 'Installing bridge and tuning CNIs'
+              cp -f /usr/src/containernetworking/plugins/bin/bridge {{ .CNIBinDir }}/cnv-bridge || exit 1
```

Comment:
What is the point of the exit? Would not it happen anyway?

Reply:
No, it will not. This is not in a bash script with -e set.

Reply:
```
The shell does not exit if ... part of any command executed in a && or ||  list  except  the  command
                      following  the final && or ||
```

Today I learned. Ack